ooyala_player/public/js/ooyala_player.js

```diff
@@ -7,6 +7,9 @@ function OoyalaPlayerBlock(runtime, element) {
             this.ccLang = this.data.ccLang;
             this.transcriptLang = null;
             this.disableCC = this.data.ccDisabled == 'True';
+            this.updateToken = null;
+            this.lastUpdate = 0;
```

Comment:
what is the usage of `lastUpdate`?

Reply:
It apparently needed to be removed. I was using it another way as I was developing. I removed it.